translations/client_ar.ts

```diff
@@ -76,18 +76,26 @@
 <context>
```

Comment:
Could you rebase on master without the translations commit :)

Reply:
that will also make CLA-Assitant happy